Orange/widgets/data/owselectrows.py

```diff
@@ -416,19 +460,11 @@ def commit(self):
                 operators = self.Operators[type(attr)]
                 opertype, _ = operators[oper_idx]
                 if attr.is_continuous:
-                    if any(not v for v in values):
+                    floats = self._values_to_floats(attr, values)
+                    if floats is None:
```

Comment:
This is ok for no values, but not if an error occurred. The old code for error (in time variable) called return.